README.md

```diff
@@ -127,6 +127,17 @@ See http://semver.org/.
 
 Please check [CONTRIBUTING](CONTRIBUTING.md) before making a contribution.
 
+## Release flow
+
+As of 2019-12, releaser is LINE's employee.
+
+Basically, versions defined in `gradle.properties`.
+
+1. Before update, update `gradle.properties` to remove `-SNAPSHOT`. e.g (`version=3.2.0`)
```

Comment:
1. Before release, updates a `version` property at `gradle.properties` with release version (without SNAPSHOT), for example `version=3.2.0`.